runtime/ftplugin/man.vim

```diff
@@ -28,8 +28,8 @@ setlocal foldcolumn=0 colorcolumn=0 nolist nofoldenable
 setlocal tagfunc=man#goto_tag
 
 if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
-  nnoremap <silent> <buffer> j             gj
-  nnoremap <silent> <buffer> k             gk
+  nnoremap <unique> <silent> <buffer> j             gj
+  nnoremap <unique> <silent> <buffer> k             gk
```

Comment:
The problem with `<unique>` is that it errors if a map already exists, which isn't what's desired when a plugin loads.